server/controllers/upload.js

```diff
@@ -96,12 +96,10 @@ exports.uploadProfilePic = asyncHandler(async (req, res, next) => {
 
 
   if (user.profilePic) {
-    const splitProfile = user.profilePic.split('https://team-honey.s3.amazonaws.com/')
-    deleteImage(splitProfile[1])
+    deleteImage(user.profilePic)
```

Comment:
Just a suggestion, maybe you can store the returned value in a variable and send it along with the response